src/components/common/Logger/Logger.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from "react";
```

Comment:
when you have a single file, to create a directory. just put the file one level above and remove

From /common/Logger/Logger.tsx to /common/Logger.tsx

Reply:
to do with other components

Reply:
issue fixed at #23